File: gnosis_wraith/server/templates/includes/code_examples.js
// JavaScript example for Gnosis Wraith API
async function crawlWebsite(url) {
    const response = await fetch('http://localhost:5678/api/crawl', {
        method: 'POST',
        headers: { 'Content-Type': 'application/json' },
        body: JSON.stringify({ url })
    });
    return await response.json();
}

// Example usage
crawlWebsite('https://example.com')
    .then(result => console.log(result))
    .catch(error => console.error(error));

File: web/templates/includes/code_examples.py
# Python example for Gnosis Wraith API integration
import requests
import json

def crawl_website(url):
    """Crawl a website using Gnosis Wraith API"""
    response = requests.post('http://localhost:5678/api/crawl', 
                           json={'url': url})
    return response.json()

# Example usage
result = crawl_website('https://example.com')
print(json.dumps(result, indent=2))